RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+This patch will minimize examples that would come out non-minimal in previous versions.
```

Comment:
Add "Thanks to Kyle Reeve for this patch." to the release note (if you're comfortable with that) - and add yourself to the list of contributors in `CONTRIBUTING.rst`!